components/gems/Rakefile

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright:: Copyright (c) 2019 Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+task :default do
+  puts "There is no default task. You probably want 'bundle exec rake update'"
+end
+
+desc "Update the One True Gemfile.lock in this directory with the latest dependencies."
+task :update do
```

Comment:
We probably want the other update tasks so we can do both the gemfile.lock and the omnibus.lock in one task

Reply:
I don't think we do in this case - it worked at the top-level of chef-dk because you were updating all deps of chef-dk. 

It doesn't fit as well here, because the dep update is occurring within a single component that's not connected to the build-time environment''s dependencies for the building package.